extensions/hibernate-orm/deployment/src/main/java/io/quarkus/hibernate/orm/deployment/HibernateOrmProcessor.java

```diff
@@ -223,7 +230,7 @@ void handleNativeImageImportSql(BuildProducer<SubstrateResourceBuildItem> resour
                 resources.produce(new SubstrateResourceBuildItem(
                         (String) i.getDescriptor().getProperties().get("javax.persistence.sql-load-script-source")));
             } else {
-                resources.produce(new SubstrateResourceBuildItem("import.sql"));
+                resources.produce(new SubstrateResourceBuildItem(getSqlLoadScript()));
```

Comment:
And with using an `Optional` (see comment below) you could manage this properly too.